models/clm/readme.rst

```diff
@@ -1,40 +1,195 @@
+     end select
+  
+     ! Since variables can use only one of these dimensions,
+     ! there is no need to check the other dimensions. 
+     if (related) exit RELATEDLOOP
+  
+  enddo RELATEDLOOP
+
+
+Snow Data Assimilation
+~~~~~~~~~~~~~~~~~~~~~~
+
+The *prognostic* variables for snow (i.e. the ones that impact the forecast) 
+are the ones that have layers. The snow observations are typically without 
+explicit depths and are essentially column-integrated quantities like snow 
+water equivalent (SWE - CLM variable *H2OSNO*) or snow depth 
+(CLM variable *SNOW_DEPTH*).  These CLM *diagnostic* variables  
+simplify the forward operator and have been part of the DART state. 
+However, updating the prognostic variables 
+(*T_SOISNO, H2OSOI_LIQ, H2oSOI_ICE, DZSNO, ZSNO, ZISNO*)
+through their ensemble covariance with the update to *H2OSNO*, for example
+will generally not result in a posterior SWE (calculated from the prognostic 
+variables) that matches the posterior SWE in *H2OSNO*. Consequently, a 
+partitition function should be used to redistribute the posterior SWE into
+the appropriate prognostic variables. **This is under development.**
+Note that we have not attempted to include any of the snow property
+variables (grain radius, carbon content, etc) in the DART state.
+
+The snow formulation in CLM is complex. Reducing the amount of snow through
+assimilation is well-defined. Creating snow when there is none 
+**is not supported** in CLM-DART. The snow-relevant column values remain 
+unchanged. **If any ensemble member does not have snow** the statistical
+assumptions for ensemble data assimilation are not valid and the snow variables
+**for all members** remain unchanged. The best method would be to alter the
+amount of snow *from the forcing file* and let CLM manage the snow. This is
+beyond the scope of CLM-DART. We have thought that if one member does not have
+snow - maybe we should just use the values from some other member - but when
+does that stop being acceptable? 10 ensemble members? 20? The distributions
+become multimodal, and the logical end result is that you could wind up using
+1 ensemble member to declare the snow for all the remaining members. That seems
+like a bad idea.
+
+The same logic applies to the variables related to plant growth. If the LAI
+observations indicates there should be something growing and nothing has
+sprouted yet ... DART essentially gives up and moves on ... 
+
+
+Configuring an Experiment
+-------------------------
+
+Our notes on how to set up, configure, build, and run CESM for an assimilation 
+experiment evolved into scripts. These scripts are not intended to be a 'black box'; 
+you will have to read and understand them and modify them to your own purpose. 
+They are heavily commented -- in keeping with their origins as a set of notes. 
+If you would like to offer suggestions on how to improve those notes - please 
+send them to dart@ucar.edu - we'd love to hear them.
```

Comment:
This next section is the section to skip ... I have not changed the table describing the scripts ...